modules/light-clients/08-wasm/keeper/keeper.go

```diff
@@ -18,15 +18,19 @@ import (
 
 	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/internal/ibcwasm"
 	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
+	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
```

Comment:
we should be able to remove this import and rely solely on the expected keeper interface added in `types`

Reply:
This is for importing errors though?

Reply:
my b 👍